docs/tutorial/intro.md

```diff
@@ -19,3 +19,7 @@ On a high level, this tutorial covers following:
 2. This [link](./volumes/snapshot.md) explains the volume snapshot related operations.
 3. This [link](./volumes/resize.md) explains the volume resize related operations.
 4. This [link](./volumes/migration.md) explains the volume migration related operations across k8s nodes.
+
+## FAQ and Troubleshooting
+
+1. My nodes in Kubernetes cluster were replaced and the cStor pools are pending. How can I fix this? See [Node Replacement](./troubleshooting/migrate_pool_by_migrating_disks.md).
```

Comment:
The link is not working. I think ../ is required in address.

Reply:
It will work after merging the PR. Verified using [this](https://github.com/openebs/cstor-operators/blob/22b7bf8a1ad03661c020f88e4b8d264f96e63dc0/docs/tutorial/intro.md) link.